gammapy/modeling/models/spatial.py

```diff
@@ -775,6 +775,28 @@ def to_region(self, **kwargs):
             **kwargs,
         )
 
+    @classmethod
+    def from_region(cls, region, **kwargs):
+        """Create a `DiskSpatialModel from a ~regions.EllipseSkyRegion`
+
+        Parameters
+        ----------
+        region : `~regions.EllipseSkyRegion`
+            region to create model from
+
+        Returns
+        -------
+        spatial_model : `~gammapy.modeling.models.DiskSpatialModel`
+        """
```

Comment:
Please add a check like `isinstance(region, EllipseSkyRegion)`...

Reply:
Done! I thought our dev docs dissuade adding instance checks https://docs.gammapy.org/1.0/development/dev_howto.html#what-checks-and-conversions-should-i-do-for-inputs